ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml

```diff
@@ -100,14 +100,14 @@
     name: products/common/hand-manual
   when: customer_key is defined and customer_key | length > 0
 
-- name: Create shared icons
-  include_tasks: shared-roles.yml
-
 - name: Set default biotacs to false if Shadow Glove is being used (user can still override)
   set_fact:
     biotacs: false
   when: glove=="shadow_glove"
```

Comment:
This can be removed right? We don't use biotacs anymore

Reply:
Do we not at all?